Documentation/teaching/labs/deferred_work.rst

```diff
@@ -853,7 +853,7 @@ section if needed.
 .. hint:: Add a new field with the type :c:type:`struct work_struct`
 	  in your device structure. Initialize this field. Schedule
 	  the work after N seconds from the timer's handler using
-	  :c:func:`schedule_work`.
+	  :c:func:`schedule_delayed_work`.
 
```

Comment:
Are you sure here we need schedule_delayed_work?

Reply:
> Schedule the work after N seconds from the timer’s handler using schedule_work().

You are right both as: it can be interpreted as scheduling from the timer handler, and the timer should be scheduled after N seconds, or you can interpret it as schedule the work to start after N seconds from the timer handler.

Could we make this any clearer from the text?

Reply:
Yes, it is confusing. I think the original intent was to schedule the timer after N seconds and from the timer to schedule the workqueue. The description it is more clear in the original lab in romanian.

I think we should add the following text which was lost in translation: " Timer-ul va fi planificat să ruleze după N secunde, rutina de tratare a timer-ului va planifica work-ul și acesta va rula cât mai aproape de acel moment."